README.md

```
@@ -333,30 +393,14 @@ To create an official release, follow these steps:
      git push origin main
      ```
 
-6. **Create GitHub Release**:
-   - Go to the GitHub repository and create a new release using the tag created in step 3. Include the release notes created in step 4.
+5. **Create GitHub Release**:
+   - Go to the GitHub repository and create a new release using the tag created in step 3.
 
-7. **Verify Release**:
-   - Verify that the release has been created successfully and that the release notes are accurate.
+6. **Verify Release**:
+   - Verify that the release has been created successfully and that all components are working as expected.
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)